packages/tailwindcss-transformer/src/__tests__/__snapshots__/transform-cva.test.ts.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`cva 1`] = `
+"const button = cva({
+      base: "cl-7e77acd4",
+      variants: {
+        intent: {
+          primary: [
+            "cl-44eeb93b",
+            "cl-5e9a40ae",
+            "cl-e9c0d2f3",
+            "cl-d8b3f644",
+          ],
+          secondary: "cl-6537e500",
+        },
+        size: {
+          small: "cl-716a7c80",
+          medium: "cl-3f348faf",
+        },
+      },
+      compoundVariants: [
+        {
+          intent: "cl-986a1b71",
+          size: "cl-c082456a",
+          className: "cl-d2cf63c7"
+        },
+      ],
+      defaultVariants: {
+        intent: "cl-986a1b71",
+        size: "cl-c082456a",
+      },
```

Comment:
These shouldn't be transformed 🙈 – I wonder if we can special-case `cva` and only transform certain keys (i.e. `base`, `variants` in this case)

Reply:
Oh shoot good catch; I completely missed that bit